backend/apps/accounts/templates/registration/login.html

```diff
@@ -34,7 +38,9 @@ <h1 class="font-weight-light">{% blocktrans %}Login{% endblocktrans %}</h1>
                 <form method="post" novalidate>
                     {% csrf_token %}
                     <input type="hidden" name="next" value="{{ next }}"> {{ form.username|as_crispy_field }} {{ form.password|as_crispy_field }}
-                    <button type="submit" class="btn blue text-white btn-md">Anmelden</button>
+                    <button type="submit" class="btn blue text-white btn-md" style="margin-top:15px;">Anmelden <svg class="bi bi-person-check" width="1em" height="1em" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Missing `blocktrans`, will change that
```suggestion
                    <button type="submit" class="btn blue text-white btn-md" style="margin-top:15px;">{% blocktrans %}Anmelden{% endblocktrans %} <svg class="bi bi-person-check" width="1em" height="1em" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```